test/distributed_ranges/include/common_tests.hpp

```diff
@@ -367,6 +362,7 @@ template <rng::range R1, rng::range R2> bool operator==(R1 &&r1, R2 &&r2) {
   return is_equal(std::forward<R1>(r1), std::forward<R2>(r2));
 }
 
+#ifdef USE_FMT
```

Comment:
why is this needed? I guess these operators are not needed at all if code compiles well without USE_FMT flag.

Reply:
Indeed. Removed

Reply:
@mateuszpn , I see it is still not removed